patch from Debian
based on patch by John Fitzgerald <jcf@wave.co.nz>
Debian-Bug: #645312
--- moc-2.5.0_alpha4_p20111211.orig/config.example.in	2012-03-31 00:39:35.534746751 +0200
+++ moc-2.5.0_alpha4_p20111211/config.example.in	2012-03-31 00:41:42.019822354 +0200
@@ -345,7 +345,7 @@
 # be assessed on content (which may be either Vorbis or Speex).
 #
 #PreferredDecoders	+= ogg(vorbis):oga(vorbis):spx(speex)
-#PreferredDecoders	+= application/ogg():audio/ogg()
+#PreferredDecoders	+= application/ogg(vorbis):audio/ogg(vorbis)
 
 # Resample method to use. There are a few methods of resampling sound supported
 # by libresamplerate. The default is the fastest (linear). Better description
diff -ur moc-2.5.0_alpha4_p20111211.orig/options.c moc-2.5.0_alpha4_p20111211/options.c
--- moc-2.5.0_alpha4_p20111211.orig/options.c	2012-03-31 00:39:36.317846278 +0200
+++ moc-2.5.0_alpha4_p20111211/options.c	2012-03-31 00:42:06.963992630 +0200
@@ -646,7 +646,7 @@
 	                 "audio/aac(aac,*):audio/aacp(aac,*):audio/m4a(ffmpeg):"
 	                 "audio/wav(sndfile,*,modplug):"
 	                 "ogg(vorbis):oga(vorbis):spx(speex):"
-	                 "application/ogg():audio/ogg()",
+	                 "application/ogg(vorbis):audio/ogg(vorbis)",
 	                 CHECK_FUNCTION);
 	add_int  ("ForceSampleRate", 0, CHECK_RANGE(1), 0, 500000);
 	add_str  ("HTTPProxy", NULL, CHECK_NONE);
